web/pom.xml

```diff
@@ -875,10 +841,211 @@
+                    </artifactItem>
+                    <artifactItem>
+                      <groupId>${project.groupId}</groupId>
+                      <artifactId>schema-csw-record</artifactId>
+                      <version>${gn.schemas.version}</version>
+                      <type>jar</type>
+                      <overWrite>false</overWrite>
+                      <outputDirectory>${basedir}/src/main/webapp/WEB-INF/data/config/schema_plugins</outputDirectory>
+                      <includes>plugin/**</includes>
+                      <fileMappers>
+                        <fileMapper implementation="org.codehaus.plexus.components.io.filemappers.RegExpFileMapper">
+                          <pattern>^plugin/</pattern><replacement>./</replacement>
+                        </fileMapper>
+                      </fileMappers>
+                    </artifactItem>
+                    <artifactItem>
+                      <groupId>${project.groupId}</groupId>
+                      <artifactId>schema-iso19110</artifactId>
+                      <version>${gn.schemas.version}</version>
+                      <type>jar</type>
+                      <overWrite>false</overWrite>
+                      <outputDirectory>${basedir}/src/main/webapp/WEB-INF/data/config/schema_plugins</outputDirectory>
+                      <includes>plugin/**</includes>
+                      <fileMappers>
+                        <fileMapper implementation="org.codehaus.plexus.components.io.filemappers.RegExpFileMapper">
+                          <pattern>^plugin/</pattern><replacement>./</replacement>
+                        </fileMapper>
+                      </fileMappers>
+                    </artifactItem>
+                    <artifactItem>
+                      <groupId>${project.groupId}</groupId>
+                      <artifactId>schema-dublin-core</artifactId>
+                      <version>${gn.schemas.version}</version>
+                      <type>jar</type>
+                      <overWrite>false</overWrite>
+                      <outputDirectory>${basedir}/src/main/webapp/WEB-INF/data/config/schema_plugins</outputDirectory>
+                      <includes>plugin/**</includes>
+                      <fileMappers>
+                        <fileMapper implementation="org.codehaus.plexus.components.io.filemappers.RegExpFileMapper">
+                          <pattern>^plugin/</pattern><replacement>./</replacement>
+                        </fileMapper>
+                      </fileMappers>
+                    </artifactItem>
+                  </artifactItems>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <!-- Copy schema resources using relative paths -->
+    <profile>
+      <id>copy-schemas</id>
+      <activation> <!-- use -Dcopy to activate       -->
+        <property><name>copy</name></property>
```

Comment:
copy is fairly generic.  i.e. -Dcopy is not that meaningful.  I would recommend that we use -Dcopy_schema or -DcopySchema of something along those lines.

Reply:
`copySchema` is now used